### PR TITLE
feat(issuer-api): allow to query certs by generationEnd frame

### PR DESCRIPTION
**Pull request by soanvig**
For 24/7 applications we can have more than 10k certificates. It would be actually really nice to have a way to query only for certificates from given time-span, instead for querying for all of them.

### CODE REVIEW

File: packages/traceability/issuer-api/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "test": "jest --env 'node'",
         "test:watch": "jest --watch",
         "test:cov": "jest --coverage",
-        "test:mocha": "mocha -r ts-node/register test/*.e2e-spec.ts --timeout 60000 --exit",
+        "test:mocha": "mocha -r ts-node/register test/certificate.e2e-spec.ts --timeout 60000 --exit",
```
Comment (josipbagaric) [inline packages/traceability/issuer-api/package.json]: ☝️

**Comment by josipbagaric** [comment]
@soanvig could you add some context on why this change is necessary?

File: packages/traceability/issuer-api/package.json (hunk 1 above)
Comment (soanvig) [reply packages/traceability/issuer-api/package.json]: done

**Comment by josipbagaric** [comment]
Could we add some other filters as well?

**Comment by soanvig** [comment]
I think we could but I'm not sure what may be useful

**Comment by josipbagaric** [comment]
I think filters by `deviceId`, `generationStartTime` (similar to endtime), and `creationTime` would be useful.

**Comment by soanvig** [comment]
> I think filters by `deviceId`, `generationStartTime` (similar to endtime), and `creationTime` would be useful.

done!

**Review by JonaszJestem — APPROVED** [review]

**Review by JonaszJestem — APPROVED** [review]

**Pull request merged**